src/delete/error.rs

```diff
@@ -3,9 +3,9 @@ use thiserror::Error;
 
 #[derive(Debug, Error)]
 pub enum DeleteError {
-    #[error("An error occurred while reading the cage config — {0}")]
-    CageConfigError(#[from] crate::config::CageConfigError),
-    #[error("No Cage Uuid given. You can provide one by using either the --cage-uuid flag, or using the --config flag to point to a Cage.toml")]
+    #[error("An error occurred while reading the enclave config — {0}")]
```

Comment:
Nitpick that was already there before this PR, but is it worth capitalising `Enclave` while we're renaming to make it consistent across all the error messages? I think there's a few in the other error enums (env/mod.rs and delete/error.rs)

Reply:
yeah that makes sense it was similar in the UI